### ISSUE TITLE
Introduce notify events for schedulerstate changes for "unresolvable" and "broken" schedulerstate

### ISSUE THREAD

**Issue by gollub** [openSUSE/open-build-service#1115]
Feature/enhancement request.

It would be nice to get notify events for scheduler scheduler state changes which result in a non-building "package" state. E.g. "unresolvable" or "broken".

To allow to write ActionMailer notify events or custom notify plugins to report those broken/unresolvable states.

**Comment by adrianschroeter** [comment]
Emitting notifications from scheduler is something what we do not so far. In any case we would need to emit the state only when the former state was different (not everytime the scheduler is checking).
Also unresolvable errors are often "okay" for a temporary time, but not really a bug.

So we should not send them out until the entire repository has finished?

**Comment by gollub** [comment]
I agree. The unresolvable one shouldn't be send out until the entire repository has finished and settled.

If there are more detailed issues in the "broken" state which are only a temporary issue ... we could apply the same logic.

**Comment by dcermak** [comment]
> Emitting notifications from scheduler is something what we do not so far. In any case we would need to emit the state only when the former state was different (not everytime the scheduler is checking).

Has this changed in the past decade? Could we get this feature eventually?
